src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age = null,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  let notMarried = '';
+
+  if (!isMarried) {
+    notMarried = 'I am not married';
+  }
```

Comment:
If you wanna do this way, don't reassign values
```suggestion
  const notMarried = isMarried ? '' : 'I am not married';
```

Reply:
Fixed, thank you!